app/screens/settings/Settings.tsx

```diff
@@ -524,12 +526,19 @@ class Settings extends Component<Props, State> {
   toggleSignMessageModal = ({ index }: { index: number }) => {
     this.setState({ signMessageModalAccountIndex: index });
   };
+
+  closeWallet = () => {
+    const { closeWallet, history } = this.props;
+    // @ts-ignore
```

Comment:
seems like no issues with this invoke, maybe remove ts-ignore

Reply:
Should not be a problem, but actually, we have a problem with the type of thunk dispatch here.
We have a lot of places with `@ts-ignore` like this one.

I've tried to get rid of it, but it requires writing some new types (aka types for each action creator), but I want to migrate to `redux-slice` (`redux-toolkit`) one day, and then we won't need it. So the idea of using this dirty thing is that we already have it in the codebase and all this stuff should be removed after migration.

If you have an idea of how to fix it easily without writing a lot of additional code and avoid using `ts-ignore` — I'm open to suggestions/discussions.

P.S.
I hope that one day we can turn on strict mode and stop writing such kludges. But there is a lot of work to do so.
It is much easier to write in strict mode from scratch :)